week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -19,24 +19,44 @@ You have to implement the missing features according to the specification.
 
 // Here is your starting point:
 let bottle = {
-  volume: 0,
-  fillUp: function() {
+  volume: 100,
+  fillUp: function () {
     // calling this function should pour your bottle full (volume = 100);
+    return (this.volume = 100);
   },

```

Comment:
no need for return inside this function because it does something it's not returning something.